requirements.txt

```diff
@@ -1,5 +1,5 @@
 # runtime
 psutil
 enum34;python_version<"3.4"
-PySide2==5.11.1
+PySide2==5.13.0
```

Comment:
is it mandatory or 5.12 can work as well?

Reply:
Yes, 5.12 is compatible.
We use 5.13 here to match the version we will use to make the release binaries.

Reply:
ok, can we add a line in `INSTALL.md` with the minimum qt version supported and warning to have a matching version between qt and pyside?
E.g. here https://github.com/alicevision/meshroom/blob/develop/INSTALL.md#python-environment adding another bullet:
```markdown
*  Qt and PySide 5.X.Y.  To install the latest supported version of PySide, you can use `requirements.txt` as described below, or you can manually install the version that matches the Qt version installed in your system.
```